easybuild/easyblocks/p/petsc.py

```diff
@@ -112,13 +112,16 @@ def configure_step(self):
             self.cfg.update('configopts', '--with-fc="%s"' % os.getenv('F90'))
 
             # compiler flags
+            # Don't build with MPI c++ bindings as this leads to a hard dependency
+            # on libmpi and libmpi_cxx even for C code and non-MPI code
+            cxxflags = os.getenv('CXXFLAGS') + ' -DOMPI_SKIP_MPICXX'
```

Comment:
The correct flag depends on which MPI is used.
OMPI_SKIP_MPICXX for OpenMPI and MPICH_SKIP_MPICXX for IMPI (and any other MPICH based MPI)
So better set both and be done with it.

Reply:
```suggestion
            cxxflags = os.getenv('CXXFLAGS') + ' -DOMPI_SKIP_MPICXX -DMPICH_SKIP_MPICXX'
```